We can't ship such strict versions of dependencies as other packages will need newer versions

diff --git a/requirements.txt b/requirements.txt
index d431540..1f1bb30 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1 +1 @@
-python-dateutil==2.6.0
\ No newline at end of file
+python-dateutil
